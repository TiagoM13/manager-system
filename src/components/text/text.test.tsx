import React from 'react';

import { render } from '@testing-library/react';

import { TextTitle } from '.';

describe('<TextTitle />', () => {
  test('should render correctly component', () => {
    const title = 'Teste';
    const { getByText, asFragment } = render(<TextTitle>{title}</TextTitle>);

    expect(getByText(title)).toBeTruthy();
    expect(asFragment()).toMatchSnapshot();
  });
});
