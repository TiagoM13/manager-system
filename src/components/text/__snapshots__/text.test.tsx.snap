// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TextTitle /> should render correctly component 1`] = `
<DocumentFragment>
  <h2
    class="sc-braxZu gtdyaj"
  >
    Teste
  </h2>
</DocumentFragment>
`;
